backend/tracim_backend/lib/webdav/dav_provider.py

```diff
@@ -58,15 +58,15 @@ def current_user(self):
         Current authenticated user if exist
         """
         return self._generate_if_none(
-            self._current_user, self._get_user, self._get_current_user_email
+            self._current_user, self._get_user, self._get_current_username
         )
 
-    def _get_user(self, user_email: typing.Callable):
-        user_email = user_email()
+    def _get_user(self, get_username: typing.Callable):
```

Comment:
Same remark for `get_username`. `get_webdavuser`?

Reply:
get login.